src/style.css

```diff
@@ -1,3 +1,95 @@
-body {
+* {
   margin: 0;
+  padding: 0;
 }
+
+:root {
+  --blue: #00ACDC;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-right: 50px;
+}
+
+header:hover {
+  border: 3px solid var(--blue);
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  width: 90px;
+  height: 60px;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.img {
+height: 40px;
+}
+
+.img:hover {
+  border: 3px solid var(--blue);
+  cursor: none;
+  cursor: default;
+
+}
+
+nav {
+  width: 641px;
+}
+
+.li {
+  text-decoration: none;
+
+}
+
+.li:hover {
+  border-bottom: 1px solid var(--blue);
+  cursor: default;
+}
+
+ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 60px;
+  align-items: center;
+}
+
+.menu {
+  list-style: none;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  color: black;
+  height: 12px;
+}
+
+.extra1 {
+  color: var(--blue);
+}
+
+.p {
+  font-weight: 400;
+}
+
+.is-active {
+  height: 12px;
+}
+
+.is-active::after {
+  display: block;
+  position: relative;
+  content: '';
+  background-color: var(--blue);
+  width: 37px;
```

Comment:
width should be 100%

Reply:
Still not fixed
<img width="885" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/b6b2ece3-ee2e-4ca9-9549-eb49f2430a95">